pkg/nificlient/client.go

```diff
@@ -204,7 +204,6 @@ func (n *nifiClient) getNifiGoApiConfig() (config *nigoapi.Configuration) {
 	if n.opts.UseSSL {
 		transport = &http.Transport{}
 		config.Scheme = "HTTPS"
-		n.opts.TLSConfig.BuildNameToCertificate()
```

Comment:
I'm guess these were pointless statements?

Reply:
`BuildNameToCertificate` has been deprecated:
> httpsServer.TLSConfig.BuildNameToCertificate has been deprecated since Go
1.14: NameToCertificate only allows associating a single certificate with a
given name. Leave that field nil to let the library select the first
compatible chain from Certificates

I think just removing this call, keeps the functionality the same?

Reply:
This is how i interpret it as well 👍